guide/russian/php/PHP Variables/variables_php.md

```diff
@@ -0,0 +1,58 @@
+---

```

Comment:
Hi @serg-t 

Please change the folder name to be lowercase and to be separated with hyphens. Also change the filename back to `index.md`.

Then also use `code blocks` for any code examples you use in the article.